tests/StatsdClient.Tests/Serializer/MetricSerializerTests.cs

```diff
@@ -50,6 +50,45 @@ public void SendIncreaseCounterByXAndSampleRateAndTags()
                 tags: new[] { "tag1:true", "tag2" });
         }
 
+        [Test]
+        public void SendCounterWithTimestamp()

```

Comment:
It validates that the timestamp is correctly appended in every situation (i.e. when there is a sample rate and when there is not, when there is tags and when there is not).

Reply:
I am not sure it is needed when I look at your code changes which just append at the end: https://github.com/DataDog/dogstatsd-csharp-client/pull/185/files#diff-9b6622ef9d4ae2b0b231e2334fb6e961f88421564541be2ae1021429795e6e1dR57-R61.

Reply:
While I agree that current code is just appending, units test are not here to test current code but to test current code and its future changes. Right now the implementation is basically appending to the end, in the future it may be different and, I want to test that there is a timestamp regardless if there is a sample rate, tags, or none of these.